### PR TITLE
add anish.is-a.dev

### PR DESCRIPTION
**Pull request by anish-mandal**
To make our job easier, please spend time to review your application before submitting. 

#### Add a x between [] if you meet that requirement example `[x]`
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. **This requirement can be raised if the domain you're registering is for emails**
  - Link to website: https://anish-mandal.github.io/
- [x] The website is reachable.
- [x] The CNAME record doesn't contain any slash.
- [x] There's enough information at the `owner` field.
   - You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.
 
 #### Feel free to join our discord server for more updates or any help related to the service :D - https://discord.gg/PZCGHz4RhQ

### CODE REVIEW

**Review by wdhdev — DISMISSED** [review]

**Review by MaskDuck — DISMISSED** [review]

File: domains/anish.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "owner": {
+    "username": "anish-mandal",
+    "email": "anishmandalme1234@outlook.com",
+    "twitter": "anishsay",
+  },
+  "description": "This website is for my personal portfolio.",
+  "repo": "https://github.com/anish-mandal/website",
+  "record": {
+    "CNAME": "anishdeveloper.netlify.app"
```
[the remark this answers is not in the document]
Comment (WilliamDavidHarrison) [reply domains/anish.json:10]: just realised that, i shouldnt have approved those changes 🤦

[the remark this answers is not in the document]
Comment (anish-mandal) [reply domains/anish.json:10]: oh ok I am changing that CNAME to a github page

**Review by wdhdev — CHANGES_REQUESTED** [review]
not enough content

**Comment by anish-mandal** [comment]
> not enough content

I will redesign the website soon and will add some content about my projects and works.

**Comment by WilliamDavidHarrison** [comment]
> > not enough content
> 
> I will redesign the website soon and will add some content about my projects and works.

We can't give you a domain until then.

**Comment by anish-mandal** [comment]
> > > not enough content
> > 
> > 
> > I will redesign the website soon and will add some content about my projects and works.
> 
> We can't give you a domain until then.

Ok that's understandable I will redesign my website and will soon come back to get the domain. Thanks for your time.

**Review by wdhdev — COMMENTED** [review]
Awaiting website redesign

**Comment by anish-mandal** [comment]
> This pull request has been marked as stale due to inactivity and will be closed. Comment anything on this PR to prevent it

Don't close it

**Comment by anish-mandal** [comment]
My site is ready.

**Review by wdhdev — APPROVED** [review]

**Comment by anish-mandal** [comment]
When will the pr be merged ? @WilliamDavidHarrison

**Comment by WilliamDavidHarrison** [comment]
> When will the pr be merged ? @WilliamDavidHarrison

Just wait for a mainter too

**Comment by anish-mandal** [comment]
> > When will the pr be merged ? @WilliamDavidHarrison
> 
> Just wait for a mainter too

Ok

**Review by wdhdev — APPROVED** [review]

**Comment by WilliamDavidHarrison** [comment]
Fix checks not running

File: domains/anish.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "owner": {
+    "username": "anish-mandal",
+    "email": "anishmandalme1234@outlook.com",
+    "twitter": "anishsay",
```
Comment (WilliamDavidHarrison) [inline domains/anish.json:5]:
```suggestion
    "twitter": "anishsay"
```

**Review by wdhdev — CHANGES_REQUESTED** [review]
Fix json

**Review by wdhdev — APPROVED** [review]

**Comment by WilliamDavidHarrison** [comment]
Sorry for late merge 😅

**Comment by anish-mandal** [comment]
> Sorry for late merge sweat_smile

Thanks for merging

**Pull request merged**